app/workers/delete_account_hierarchy_worker.rb

```diff
@@ -21,4 +20,12 @@ def batch_description
     end
   end
 
+  # :reek:NilCheck
```

Comment:
wouldn't be better to make `destroyable_association?` a protected method and rewrite that logic? it could be nice one liner..

Reply:
and no additional `loop` would be necessary.

Reply:
ah right :+1: good one :+1: